### ISSUE TITLE
Panel Component

### ISSUE THREAD

**Issue by oliveiraddb** [appnexus/lucid#120]

**Comment by jondlm** [comment]
Hey @oliveiraddb, could you give us a little more context as to what you're thinking? We like to get UX in front of us before we start building new components in Lucid so they fit into the larger framework in a meaningful way.

**Comment by oliveiraddb** [comment]
Oh sorry. I'm suggesting a Panel like this one:
http://getbootstrap.com/components/#panels
I think it'd be good to have information organized in blocks when
necessary. It could have properties like title and children components like
Header, Body and Footer.

If you guys like the ideia but think it has low priority, let me know so
maybe i can create PR with that component.



2016-03-17 18:41 GMT-03:00 Jon de la Motte <notifications@github.com>:

**Comment by jondlm** [comment]
No worries! Hey @halseyhemingway is there anything on the roadmap that a need for something like this?

**Comment by halseyhemingway** [comment]
Yes it is! Its on JIRA as ANXR-258 (the UX team has been calling it a "container"). Its somewhere in the middle of the backlog at the current moment. Let me see if David can get us a quick mock. It should be pretty easy to knock out.

**Comment by jondlm** [comment]
Awesome, sounds good. @oliveiraddb that'd be awesome if you want to tackle this once the mock up is ready. Make sure to read over the [contributing docs][cont] and let me know if you have any questions about implementation/coding style.

[cont]: https://github.com/appnexus/lucid/blob/master/CONTRIBUTING.md

**Comment by oliveiraddb** [comment]
Great! I can do that!

If needed, I can handle issue #112 (Modal component) too as I see Modal as a Panel (container) with a close button and a layer behind. What do you guys think?

**Comment by jondlm** [comment]
@oliveiraddb I'm actually almost done with `Modal`, there will be a PR for it later this afternoon :) And you're correct, they will look pretty similar likely.
